package.json

```diff
@@ -69,9 +69,9 @@
   "man": "",
   "scripts": {
     "eslint": "eslint index.js lib bin examples --quiet && eslint --config ./test/.eslintrc test/ --quiet",
-    "mocha-coverage": "nyc --reporter=html mocha test/src/",
-    "test": "mocha test/src/",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "mocha-coverage": "npx nyc --reporter=html mocha test/src/",
```

Comment:
`npx` is redundant here, as far as I know.

Reply:
nyc 'not found' errors if npx is not used which will refer the executables from node_modules/.bin folder. Or we have to install nyc globally